util/scrape.py

```diff
@@ -0,0 +1,70 @@
+#Scrapes foosball rules from the official foosball website
+#and produces a rules.json file.
+
+from bs4 import BeautifulSoup
+import json
+import requests
+import re
+
+class Scraper:
+
+    def parse_long(self):
+        self.setup_soup("http://www.foosball.com/learn/rules/ustsa/")
+        return self.__dfs(self.soup.ol)
+
+    def parse_short(self):
+        self.setup_soup("http://www.foosball.com/learn/rules/onepage/")
+        return self.__short()
+
+    def setup_soup(self, url):
+        html = requests.get(url).text
+        self.soup = BeautifulSoup(html, 'lxml')
+
+    #perform a dfs and make a list of lists representing a tree of
+    #rules
+    def __dfs(self, node):
+        if len(node.contents) <= 1:
+            return node.text.replace("\n", "").replace("\r", "")
+        else:
+            return [self.__dfs(i) for i in node.findChildren()]
+
+    #implementation of the short parse method
+    def __short(self):
+        table = self.__get_tables()
+        rule_dict = self.__build_rule_dict(table)
+        self.__write_rules(rule_dict)
+
+    #Gets parses out both tables and divides them based on rule number
+    def __get_tables(self):
+        tables = self.soup.findAll("td")
+        table_one = re.split("[0-9]+\.", tables[0].text)
+        table_two = re.split("[0-9]+\.", tables[1].text)
+        table_one.extend(table_two)
+        return table_one
+
+    #Given the the table html from get_tables, build up the rule dictionary
```

Comment:
I find it to be more difficult to read comments that don't have a space between the `#` and the comment. Is there any benefit to doing it this way?